### DISCUSSION TITLE
Responsive View

### DISCUSSION THREAD

**Discussion by MonMohon** [nolimits4web/swiper#5242] [Q&A] [answered]
i'm using the following code it works fine for desktop but how can I show 1/slidesPerView={1} item on mobile
`
<Swiper
      spaceBetween={50}
      pagination={{ "clickable": true }}
      slidesPerView={3}
      navigation={true}
    >
`
please help me

**Comment by vltansky** [comment, marked as answer]
Try `<Swiper slidesPerView={1} breakpoints={{768: { slidesPerView: 3}}} />`

**Reply by dancerist** [reply]
Hi, I'm having same request. I'd like to show 1 item in smaller screen. Where do I add that code snippet?

**Reply by dancerist** [reply]
![image](https://user-images.githubusercontent.com/51177627/152127357-16a84ff6-5613-470a-b25a-756ee38070ba.png)

**Reply by martinscola-hwm** [reply]
It goes inside the main config object, 'breakpoints' is at the same level as 'slidesPerView', loop, etc. Hope it helped

**Comment by ArslanIrsalAlpha** [comment]
how can i use breakpoints on swiper element please help
<swiper-container slides-per-view="3" navigation="true" scrollbar="true" pagination-dynamic-bullets="true" autoplay-delay="2500" autoplay-disable-on-interaction="false" pagination-clickable="true" breakpoints="???">

is there any way to include in it here instead of the init="false" way ?
